internal/oci/container.go

```diff
@@ -390,14 +390,9 @@ func (c *Container) exitFilePath() string {
 
 // IsAlive is a function that checks if a container's init PID exists.
 // It is used to check a container state when we don't want a `$runtime state` call
-func (c *Container) IsAlive() bool {
+func (c *Container) IsAlive() error {
```

Comment:
Maybe it's me and my mis-spent youth doing Java, but I don't think any function that starts with "is" should return anything other than a boolean.  I'd rename the function.

Reply:
I will look into this as a follow up in the master version :)